cheroot/test/test_core.py

```diff
@@ -195,7 +195,8 @@ def test_parse_uri_invalid_uri(test_client):
     response = _get_http_response(c, method='GET')
     response.begin()
     assert response.status == HTTP_BAD_REQUEST
-    assert response.read(21) == b'Malformed Request-URI'
+    # TODO: h11 doesn't have this specific of messaging. See python-hyper/h11#98
+    # assert response.read(21) == b'Malformed Request-URI'
```

Comment:
@ian-otto I think this can be corrected now

Reply:
Yes and no. My PR over in that repo only handled updating the messaging for a few specific cases that I felt were very important to be distinct. That said, h11 still gives fairly generic errors when it comes to request line problems. Part of me thinks that in that case, it may be okay to let that happen, since request-line issues are generally pretty easy to spot. I'm curious on your opinion though.

Reply:
I think that we could leave it as-is for now and maybe add that in the future. OTOH it'd nice UX/DX-wise to have precise messages in general.